From d69c680903eff251e987d90af729992d2e141f9c Mon Sep 17 00:00:00 2001
From: Brian Ketelsen <bketelsen@gmail.com>
Date: Tue, 16 Jul 2024 11:13:20 -0400
Subject: [PATCH] Change RunDir file mode to 0711

This change updates the file mode of `RunDir` to 0711 allowing non-root
clients to connect to the unix socket.

Fixes #1003

Signed-off-by: Brian Ketelsen <bketelsen@gmail.com>
---
 internal/server/sys/fs.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/internal/server/sys/fs.go b/internal/server/sys/fs.go
index a179828d0b..b8afb5ecf2 100644
--- a/internal/server/sys/fs.go
+++ b/internal/server/sys/fs.go
@@ -50,7 +50,7 @@ func (s *OS) initDirs() error {
 		{filepath.Join(s.VarDir, "images"), 0700},
 		{s.LogDir, 0700},
 		{filepath.Join(s.VarDir, "networks"), 0711},
-		{s.RunDir, 0700},
+		{s.RunDir, 0711},
 		{filepath.Join(s.VarDir, "security"), 0700},
 		{filepath.Join(s.VarDir, "security", "apparmor"), 0700},
 		{filepath.Join(s.VarDir, "security", "apparmor", "cache"), 0700},
